commodore/inventory/parameters.py

```diff
@@ -259,17 +293,31 @@ def cloud_regions(self) -> Dict[str, Iterable[str]]:
         return self._cloud_regions
 
     @classmethod
-    def _make_directories(cls, work_dir: Path):
+    def _make_directories(cls, work_dir: Path, fake_tenant=True):
```

Comment:
Very much personal preference, but IMO this helper function does nothing to improve readability. It's used once and `from_repo_dirs` needs to work with the classes directory anyway

Reply:
I think I added this in #383 because I thought it might be useful when implementing the functionality for cloning global or tenant repos in the command (which still isn't there).

I've inlined it for now.